packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -106,6 +106,17 @@ class _RenderExclusiveMouseRegion extends RenderMouseRegion {
 /// the child that Tooltip wraps is hovered over on web or desktop. On mobile,
 /// the tooltip is shown when the widget is long pressed.
 ///
+/// This tooltip will default to showing above the [Text] instead of below
+/// because its ambient [TooltipThemeData.preferBelow] is false.
+/// (See the use of [MaterialApp.theme].)
+/// Setting that piece of theme data is recommended to avoid having a finger or
+/// cursor hide the tooltip. For other ways to set that piece of theme data see:
+///
+/// * [Theme.data], [ThemeData.tooltipTheme]
+/// * [TooltipTheme.data]
+///
+/// or it can be set directly on each tooltip with [Tooltip.preferBelow].
+///
 /// ** See code in examples/api/lib/material/tooltip/tooltip.0.dart **
 /// {@end-tool}
 ///
```

Comment:
In the next example (`tooltip.0.dart`), the example is meant to illustrate the panoply of options available, but it now ends up with the same effective `preferBelow` value as the other examples.

Perhaps the cleanest thing there would be to have it say `preferBelow: false` up in a `ThemeData` at the top (just like the other examples — and so, like them, setting the example that people should typically follow when not constrained by an existing codebase), and then `preferBelow: true` on the individual `Tooltip`, so that it effectively demos the option.

Reply:
> In the next example (`tooltip.0.dart`) […]

I think you mean `tooltip.1.dart`?